src/Tribe/Views/V2/Views/Reflector_View.php

```diff
@@ -18,6 +18,15 @@
  */
 class Reflector_View extends View {
 
+	/**
+	 * Slug for this view.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $slug = 'reflector';
```

Comment:
If you add `publicly_visible` to the parent class as `true`, you'll need to declare it as `false` here.